File: Recursion/JS/005.js
// print sum of 1 to N

function fun(i,sum){
    if(i<1){
       console.log(sum);
       return;
    }
    
    fun(i-1, sum+i);
 }
 
 let i = 5;
 let sum=0;
 fun(i,sum)

File: Recursion/CPP/005.cpp
// print sum of 1 to N

#include<bits/stdc++.h>
using namespace std;


void fun(int i, int sum){
    if(i<1){
        cout<<sum;
        return;
    }

    fun(i-1, sum+i);
}
int main(){
    int n;
    int sum=0;
    cin>>n;
    fun(n,sum);

}